tests/ut/test_base.py

```diff
@@ -260,64 +260,77 @@ async def test_alt_build_key_default_namespace(self, namespace, expected):
-            _get.assert_called_once_with(
-                expected, _conn=None, encoding=cache.serializer.encoding)
-
-    async def _assert_multi_get__alt_build_key_default_namespace(self, cache, expected):
-        with patch.object(cache, "_multi_get", autospec=True) as _multi_get:
-            await cache.multi_get([Keys.KEY])
-            _multi_get.assert_called_once_with(
-                [expected], _conn=None, encoding=cache.serializer.encoding)
-
-    async def _assert_set__alt_build_key_default_namespace(self, cache, expected):
-        with patch.object(cache, "_set", autospec=True) as _set:
-            await cache.set(Keys.KEY, "value")
-            _set.assert_called_once_with(
-                expected, "value", _conn=None, ttl=None, _cas_token=None)
-
-    async def _assert_multi_set__alt_build_key_default_namespace(self, cache, expected):
-        with patch.object(cache, "_multi_set", autospec=True) as _multi_set:
-            await cache.multi_set([(Keys.KEY, "value")])
-            _multi_set.assert_called_once_with(
-                [(expected, "value")], _conn=None, ttl=None)
-
-    async def _assert_exists__alt_build_key_default_namespace(self, cache, expected):
-        with patch.object(cache, "_exists", autospec=True) as _exists:
-            await cache.exists(Keys.KEY)
-            _exists.assert_called_once_with(expected, _conn=None)
-
-    async def _assert_increment__alt_build_key_default_namespace(self, cache, expected):
-        with patch.object(cache, "_increment", autospec=True) as _increment:
-            await cache.increment(Keys.KEY)
-            _increment.assert_called_once_with(expected, delta=1, _conn=None)
-
-    async def _assert_delete__alt_build_key_default_namespace(self, cache, expected):
-        with patch.object(cache, "_delete", autospec=True) as _delete:
-            await cache.delete(Keys.KEY)
-            _delete.assert_called_once_with(expected, _conn=None)
-
-    async def _assert_expire__alt_build_key_default_namespace(self, cache, expected):
-        with patch.object(cache, "_expire", autospec=True) as _expire:
-            await cache.expire(Keys.KEY, 0)
-            _expire.assert_called_once_with(expected, 0, _conn=None)
+        BaseCacheBuildKeyMethods = self.BaseCacheBuildKeyMethods
+        await BaseCacheBuildKeyMethods.assert_add(cache, expected)
+        await BaseCacheBuildKeyMethods.assert_get(cache, expected)
+        await BaseCacheBuildKeyMethods.assert_multi_get(cache, expected)
+        await BaseCacheBuildKeyMethods.assert_set(cache, expected)
+        await BaseCacheBuildKeyMethods.assert_multi_set(cache, expected)
+        await BaseCacheBuildKeyMethods.assert_exists(cache, expected)
+        await BaseCacheBuildKeyMethods.assert_increment(cache, expected)
+        await BaseCacheBuildKeyMethods.assert_delete(cache, expected)
+        await BaseCacheBuildKeyMethods.assert_expire(cache, expected)
+
+    class BaseCacheBuildKeyMethods:
```

Comment:
I don't see what benefit a nested class adds here. Sticking to methods on the existing class (or just global functions as it doesn't appear to need a reference to the instance anyway) seems better.

Reply:
ok, fair enough. 

I find grouping these related methods into a class cleaner, but I think it comes down to individual preferences. Let's skip this PR then.  :)

Reply:
Alright, I wouldn't be opposed to grouping into a module or something. But, not really important here.

I tend to question any use of @staticmethod, as per https://wemake-python-styleguide.readthedocs.io/en/latest/pages/usage/violations/oop.html#wemake_python_styleguide.violations.oop.StaticMethodViolation